Qubit Server/dev/src/org/labrad/qubits/QubitContext.java

```diff
@@ -568,11 +570,17 @@ public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
 	}
 	@SettingOverload
 	public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
-			@Accepts("*c") Data vals,
+			@Accepts({"*c", "(*i{I}, *i{Q})"}) Data vals,
 			boolean deconvolve) {
 		IqChannel ch = getChannel(id, IqChannel.class);
-		ComplexArray c = ComplexArray.fromData(vals);
-		ch.addData(new IqDataTime(c, !deconvolve));
+        if (vals.isCluster()) {
+            Preconditions.checkArgument(!deconvolve, "Must not deconvolve if providing DAC'ified IQ data.");
```

Comment:
indentation. can't tell whether it's tabs or spaces here, but should be 2 spaces.